wordcount.c

```diff
@@ -0,0 +1,30 @@
+/* Mariposa Lee, reads in userinput to write words to text.txt then counts the amount of words by using the amount of spaces. I researched quite a bit for the counting spaces part and mashed in a bunch of ideas together.*/
```

Comment:
this outputted an `abort trap: 6` error, which has to do with writing to memory that you do not own. Look here: http://stackoverflow.com/questions/26431147/abort-trap-6-error-in-c